Makefile

```
@@ -0,0 +1,64 @@
+	@echo "Formatting code..."
+	@cargo fmt --manifest-path ./Cargo.toml --all -- --check --unstable-features
+
+lint:
+	@echo "Linting code..."
+	@cargo clippy --manifest-path ./Cargo.toml --all-features --workspace -- -D warnings
+
+miri:
+	@echo "Running miri..."
+	@cargo miri test --manifest-path ./Cargo.toml --all-features --workspace
+
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (8).

(maxbodylength)